### PR TITLE
Fix docker images

### PR DESCRIPTION
**Pull request by muellerzr**
# What does this PR do?

Docker image workflows have been broken for the last few months silently, it comes from the workflow we use for it. Instead now we just manually upload it using the repository secrets, and I've confirmed this works and runs without issue. 

cc @LysandreJik @BenjaminBossan

### CODE REVIEW

**Comment by BenjaminBossan** [comment]
Just for context, newer versions of `docker/build-push-action` would not work or do you prefer to avoid it completely because of the observed issues?

**Comment by muellerzr** [comment]
Prefer to avoid completely so we have control over it.

File: .github/workflows/build-docker-images-release.yml [hunk 1]
```diff
@@ -35,11 +35,9 @@ jobs:
...
 
       - name: Build and Push CPU
-        uses: docker/build-push-action@v2
-        with:
-          context: ./docker/accelerate-cpu
-          push: true
-          tags: huggingface/accelerate-cpu:${{needs.get-version.outputs.version}}
+        run: |
+          docker build -t huggingface/accelerate-cpu -f ./docker/accelerate-cpu/Dockerfile .
+          docker push huggingface/accelerate-cpu::${{needs.get-version.outputs.version}}
```
Comment (BenjaminBossan) [inline .github/workflows/build-docker-images-release.yml:40]: Sorry, my docker knowledge is not deep enough: What does the `::` do? `:` would just be tag, right?

Comment (muellerzr) [reply .github/workflows/build-docker-images-release.yml]: Yes, small typo, probably from copy/paste

Comment (muellerzr) [reply .github/workflows/build-docker-images-release.yml]: (Though didn't seem to cause any issues)

**Review by BenjaminBossan — APPROVED** [review]
Thanks for clarifying. LGTM and big thanks for figuring out the issue.

**Pull request merged**